### PR TITLE
Add batch.com/headers-bundle recipe

### PR DESCRIPTION
**Pull request by nesk**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by nesk** [comment]
@Nyholm Hi! Is there anything preventing a merge?

File: batch.com/headers-bundle/1.0/config/packages/batch_headers.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        #
+        #     - name: Access-Control-Allow-Origin
+        #       value: "*"
+        #       condition: request.getPathInfo() matches '^/api'
+        #
+        # Always cache images:
+        #
+        #     - name: Cache-Control
+        #       value: max-age=31536000, public
+        #       condition: response.headers.get('Content-Type') matches '^image/'
```
Comment (Nyholm) [inline batch.com/headers-bundle/1.0/config/packages/batch_headers.yaml:19]: Examples should be in your bundle's readme. Not as comments. 

Please remove these and add a link to your bundle's docs. 

(Maybe, it makes sense to uncomment one of them an remove the others)

**Review by Nyholm — CHANGES_REQUESTED** [review]
Just a small thing

File: batch.com/headers-bundle/1.0/config/packages/batch_headers.yaml (hunk 1 above)
Comment (nesk) [reply batch.com/headers-bundle/1.0/config/packages/batch_headers.yaml]: Sorry for the delay… I've just fixed this. 🙂

**Comment by Nyholm** [comment]
@symfony-flex-server review please

**Comment by Nyholm** [comment]
Im okey with adding an empty config file for this recipe because the 100% of the users will add their headers. 

Can you make sure the CI is green?

**Comment by nesk** [comment]
@symfony-flex-server review please

**Comment by nesk** [comment]
Honestly, I can't explain why it fails here... 

```
    batch.com/headers-bundle/1.0/config/packages/batch_headers.yaml
    Unable to read the file
    batch.com/headers-bundle/1.0/manifest.json
    Unable to read the file
```

**Comment by Nyholm** [comment]
@symfony-flex-server review please

**Comment by Nyholm** [comment]
Oh.. Sorry. Can you rebase your PR to get the new validation?

**Comment by nesk** [comment]
Well, I tried, but the automatic review fails and I can't bother anymore with any new changes, I'm fed up. This common repository is a nightmare, it's been months since I tried to merge my own config and all the reviews took weeks/months…

I wish we could simply provide a custom config in our own repositories without validation from the Symfony team, these are not "contrib" recipes since we still have to pass official validation.

Nothing agains't you @Nyholm, I think you are working on this repository when you have free time, you do what you can.

**Pull request closed without merging**